t/t3303-system-healthcheck.t

```diff
@@ -65,6 +65,11 @@ test_expect_success 'overlay status is full' '
 	test "$(flux overlay status --timeout=0 --summary)" = "full"
 '
 
+test_expect_success 'flux overlay errors prints nothing' '
```

Comment:
Commit message typo: "ProbleM"

Reply:
Thanks - I forgot the man page.  Will get that done, thanks.

Reply:
OK pushed that and the typo fix and I'll set MWP.